Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -231,6 +231,10 @@
 )
 reconstruction_ecalOnly = cms.Sequence(reconstruction_ecalOnlyTask)
 
+from Configuration.Eras.Modifier_phase2_ecal_devel_cff import phase2_ecal_devel
+phase2_ecal_devel.toReplaceWith(ecalRecHitNoTPTask, ecalRecHitNoTPTask.copyAndExclude([ecalPreshowerRecHit]))
```

Comment:
era modifiers have to be applied at the place that defines the object. `ecalRecHitNoTPTask` is not defined in this file.

Reply:
It is in fact already at the right place as well. Will remove this one.